
---
- name: Test Loop Error Handling
  hosts: all
  gather_facts: no

  tasks:
  
  - name: Ping hosts
    ping:
  
  - name: Print failed hosts
    debug:
      msg: "Host {{ item }} failed to respond"
    when: "ping.results[item]['failed'] == true"
    with_sequence: start=0 end={{ groups['all'] | length - 1 }}
